### ISSUE TITLE
[WinError 216]

### ISSUE THREAD

**Issue by omergunal** [mitre/caldera#61]
I am getting this error when i start operation
Agent's operating system: Windows 7 pro

```
  {
    "_id": "5b712bef8f49c6000d4b854d",
    "agent": "5b7126b58f49c600014b854b",
    "action": {
      "create_process": {
        "process_args": "C:\\commander.exe -f",
        "parent": null,
        "hide": true,
        "output": false
      },
      "error": "agents exception",
      "exception": "Traceback (most recent call last):\n  File \"C:\\Gitlab Runner\\builds\\f8e514af\\0\\caldera\\caldera-agent\\caldera_agent\\async_client.py\", line 117, in run_job\n  File \"C:\\Gitlab Runner\\builds\\f8e514af\\0\\caldera\\caldera-agent\\caldera_agent\\interfaces.py\", line 84, in run\n  File \"C:\\Gitlab Runner\\builds\\f8e514af\\0\\caldera\\caldera-agent\\caldera_agent\\interfaces.py\", line 159, in _create_process\n  File \"C:\\Gitlab Runner\\builds\\f8e514af\\0\\caldera\\caldera-agent\\caldera_agent\\foster3.py\", line 861, in __init__\n  File \"C:\\Gitlab Runner\\builds\\f8e514af\\0\\caldera\\caldera-agent\\caldera_agent\\foster3.py\", line 1153, in _execute_child\nOSError: [WinError 216] This version of %1 is not compatible with the version of Windows you're running. Check your computer's system information to see whether you need a x86 (32-bit) or x64 (64-bit) version of the program, and then contact the software publisher\n"
    },
    "create_time": "2018-08-13T06:57:51.416Z",
    "status": "failed"
  }
```

**Comment by omergunal** [comment]
Also i have tried on windows 10 and i got different error

```
{
    "_id": "5b716d358f49c6000d4b8579",
    "agent": "5b716c8f8f49c600014b8578",
    "action": {
      "create_process": {
        "process_args": "C:\\commander.exe -f",
        "parent": null,
        "hide": true,
        "output": false
      },
      "error": "agents exception",
      "exception": "Traceback (most recent call last):\n  File \"C:\\Gitlab Runner\\builds\\f8e514af\\0\\caldera\\caldera-agent\\caldera_agent\\async_client.py\", line 117, in run_job\n  File \"C:\\Gitlab Runner\\builds\\f8e514af\\0\\caldera\\caldera-agent\\caldera_agent\\interfaces.py\", line 84, in run\n  File \"C:\\Gitlab Runner\\builds\\f8e514af\\0\\caldera\\caldera-agent\\caldera_agent\\interfaces.py\", line 159, in _create_process\n  File \"C:\\Gitlab Runner\\builds\\f8e514af\\0\\caldera\\caldera-agent\\caldera_agent\\foster3.py\", line 861, in __init__\n  File \"C:\\Gitlab Runner\\builds\\f8e514af\\0\\caldera\\caldera-agent\\caldera_agent\\foster3.py\", line 1153, in _execute_child\nOSError: [WinError 1392] The file or directory is corrupted and unreadable\n"
    },
    "create_time": "2018-08-13T11:36:21.326Z",
    "status": "failed"
  }
```

**Comment by hf-mitre** [comment]
@omergunal, about the first error message: is the Windows 7 Pro box running the 32-bit OS? Currently, we only provide x64 binaries for the agent.  

We can talk about the second error message in #59 (I saw that you commented over there as well).

**Comment by omergunal** [comment]
Its ok now. Solved in there #59
